sphinx/templates/quickstart/make.ps1_t

```diff
@@ -0,0 +1,198 @@
+# .SYNOPSIS
+# Please use '.\make.ps1 -<target>' where <target> is one of:
+#   html, dirhtml, singlehtml, pickle, json, linkcheck
+# 
+# .DESCRIPTION
+# This is a front-end script for Sphinx.
+# 
+# .EXAMPLE
+#  .\make.ps1 -clean
+# Cleanup the build directory.
+# .EXAMPLE
+#  .\make.ps1 -html
+# Build documentation as HTML pages.
+# .EXAMPLE
+#  .\make.ps1 -clean -html -linkcheck
+# Combine multiple commands in a single call
+# 
+# 
+# ==========================================================================
+# Copyright © 2016 Patrick Lehmann - Dresden, Germany
```

Comment:
(required) Please remove copyright.
After merging this PR, this ps1 file will be used for each user documentation projects. IMO, we should not make copyright notices that will cause confusion for users. Instead, please list the name as a contributor in the /AUTHORS file.